### ISSUE TITLE
always installs latest version

### ISSUE THREAD

**Issue by apaleslimghost** [jorgebucaran/nvm.fish#112]
no matter what version i ask for, `nvm` ignores it and installs `14.7`. i installed it with Fundle. am i doing something wrong?

```
~ ⟩ nvm use 10
fetching https://nodejs.org/dist/v14.7.0/node-v14.7.0-darwin-x64
#########                                                    7.9%^C⏎

~ ⟩ nvm use 10.22.0
fetching https://nodejs.org/dist/v14.7.0/node-v14.7.0-darwin-x64
#########                                                    7.9%^C⏎

~ ⟩ nvm use dubnium
fetching https://nodejs.org/dist/v14.7.0/node-v14.7.0-darwin-x64
#########                                                    7.7%^C⏎

~ ⟩ echo 10.22.0 > .nvmrc

~ ⟩ nvm
fetching https://nodejs.org/dist/v14.7.0/node-v14.7.0-darwin-x64
#########                                                    7.7%^C
```

**Comment by jorgebucaran** [comment]
@apaleslimghost I couldn't repro this here. 


<img width="1440" alt="スクリーンショット 2020-08-02 10 46 28" src="https://user-images.githubusercontent.com/56996/89113550-a58af700-d4ad-11ea-823e-7dc3026c968b.png">


1) I doubt Fundle could be part of the problem, but you may try with Fisher just to discard that possibility.
2) Please state your OS, fish, and fish-nvm version. 
3) Do you think there may be anything fishy in your `~/.config/fish/config.fish` that could be interfering with this? 
4) Please run `la ~/.config/nvm/`, `cat ~/.config/nvm/version`, and `cat ~/.config/nvm/index` and let me know what you have. 
5) Finally, do you remember having installed any utilities that replace your OS default core utilities, e.g., `awk`.

**Comment by jorgebucaran** [comment]
@apaleslimghost I just released [2.0](https://github.com/jorgebucaran/nvm.fish/releases/tag/2.0.0). This should be fixed now.
